module2_solution/app.js

```diff
@@ -0,0 +1,44 @@
+/* global angular */
+
+(function() {
+	'use strict';
+
+	angular.module('ShoppingListCheckOff', [])
+		.controller('ToBuyController', ToBuyController)
+		.controller('AlreadyBoughtController', AlreadyBoughtController)
+		.service('ShoppingListCheckOffService', ShoppingListCheckOffService);
```

Comment:
used jshint plugin of sublime text 3 which shows double spaced indentation but it does not seem persist to a different editor. Also converted tabs to spaces. 
<img width="530" alt="screen shot 2017-02-02 at 12 42 49 am" src="https://cloud.githubusercontent.com/assets/6377946/22542434/9c052b5e-e8e0-11e6-84dd-125429500d79.png">